### PR TITLE
Update iacmData.ts

### PR DESCRIPTION
**Pull request by urischeiner**
Update IaCM Roadmap

Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: src/components/Roadmap/data/iacmData.ts [hunk 1]
```diff
@@ -121,4 +91,44 @@ export const IacmData: Horizon = {
...
+      },
+      {
+        tag: [{ value: "Security" }],
+        title: "OPA integration",
+        description: "IaCM natively integrates with OPA, allowing users to define rules that can be enforced at design and runtime, eliminating policy violations and reducing the risks of security and unplanned cost",
+      },
+      {
+        tag: [],
+        title: "Cost estimation",
+        description: "Using Infracost, IaCM provides cost estimation on resource changes, allowing users to understand how cloud bill will be changed, based on resource changes",
```
Comment (RickCraig) [inline src/components/Roadmap/data/iacmData.ts:110]:
```suggestion
        description: "By utilizing Infracost, IaCM offers cost predictions for resource modifications, enabling users to foresee adjustments to their cloud expenses ahead of time.",
```

Just a suggestion, but it needs to change as the original doesn't make sense.

File: src/components/Roadmap/data/iacmData.ts [hunk 2]
```diff
@@ -121,4 +91,44 @@ export const IacmData: Horizon = {
...
+      },
+      {
+        tag: [{ value: "Security" }],
+        title: "State management",
+        description: "IaCM supports hosted backend, allowing users to store state securely",
```
Comment (RickCraig) [inline src/components/Roadmap/data/iacmData.ts:115]:
```suggestion
        description: "IaCM supports hosted remote backends, enabling users to securely store state.",
```

File: src/components/Roadmap/data/iacmData.ts [hunk 3]
```diff
@@ -121,4 +91,44 @@ export const IacmData: Horizon = {
...
+      },
+      {
+        tag: [{ value: "Security" }],
+        title: "RBAC",
+        description: "IaCM supports granular RBAC operations, where admins can define the access privileges for different roles",
+      },
+      {
+        tag: [{ value: "Developer Experience" }],
+        title: "PR Automation",
+        description: "IaCM support PR Automation that populates plan and cost data to the PR process, helping users to reduce testing time on Terraform changes",
```
Comment (RickCraig) [inline src/components/Roadmap/data/iacmData.ts:130]:
```suggestion
        description: "IaCM supports PR Automation that populates plan and cost data as a pull request comment, helping users to reduce testing time on Terraform changes",
```

Just a suggestion, needs the s after support though

**Review by RickCraig — CHANGES_REQUESTED** [review]

File: src/components/Roadmap/data/iacmData.ts [hunk 4]
```diff
@@ -66,23 +26,18 @@ export const IacmData: Horizon = {
...
-        title: "Harness CD integration",
-        description: "Integration with CD allows IaCM users to link Resources with Services and Environments, giving end-to-end visibility",
-      },
-      {
-        tag: [],
-        title: "Harness CCM Integration",
-        description: "Integration with CCM allows IaCM users to have full visibility of costs associated with the resources being managed by IaCM",
+        tag: [{ value: "Security" }],
+        title: "Security Plugins",
+        description: "Plugins for tfsec, tflint, and checkov that can be added to the IaCM pipeline to find security vulnerabilities and misconfiguration",
```
Comment (richardblack-Harness) [inline src/components/Roadmap/data/iacmData.ts:40]:
```suggestion
        description: "Add Plugins for tfsec, tflint, and checkov to your IaCM pipeline to find security vulnerabilities and misconfiguration.",
```

File: src/components/Roadmap/data/iacmData.ts [hunk 5]
```diff
@@ -94,6 +49,21 @@ export const IacmData: Horizon = {
...
+    feature: [
+      {
+        tag: [],
+        title: "Harness CD integration",
+        description: "Integration with CD allows IaCM users to link Resources with Services and Environments, giving end-to-end visibility",
+      },
+      {
+        tag: [],
+        title: "Harness CCM Integration",
+        description: "Integration with CCM allows IaCM users to have full visibility of costs associated with the resources being managed by IaCM",
```
Comment (richardblack-Harness) [inline src/components/Roadmap/data/iacmData.ts:65]:
```suggestion
        description: "Integrate with CCM to grant IaCM users full visibility of costs associated with resources managed by IaCM.",
```

File: src/components/Roadmap/data/iacmData.ts [hunk 6]
```diff
@@ -121,4 +91,44 @@ export const IacmData: Horizon = {
       },
     ],
   },
+    Released: {
+    description: "What has been released",
+    feature: [
+      {
+        tag: [{ value: "Simplify Onboarding" }],
+        title: "Customized pipeline",
+        description: "IaCM supports highly customized pipelines that allow users to create advanced workflow, add 3rd party plugins, run steps in parallel, and more.",
```
Comment (richardblack-Harness) [inline src/components/Roadmap/data/iacmData.ts:100]:
```suggestion
        description: "IaCM supports highly customized pipelines that allow users to create advanced workflows, add 3rd party plugins, run steps in parallel, and more.",
```

File: src/components/Roadmap/data/iacmData.ts [hunk 7]
```diff
@@ -121,4 +91,44 @@ export const IacmData: Horizon = {
...
+      },
+      {
+        tag: [],
+        title: "Cost estimation",
+        description: "By utilizing Infracost, IaCM offers cost predictions for resource modifications, enabling users to foresee adjustments to their cloud expenses ahead of time.",
+      },
+      {
+        tag: [{ value: "Security" }],
+        title: "State management",
+        description: "IaCM supports hosted remote backends, enabling users to securely store state.",
```
Comment (richardblack-Harness) [inline src/components/Roadmap/data/iacmData.ts:115]:
```suggestion
        description: "IaCM supports hosted remote backends, enabling users to store their infrastructure state securely.",
```

**Review by richardblack-Harness — CHANGES_REQUESTED** [review]
Couple of very minor suggestions. Otherwise LGTM!

**Comment by ravilach** [comment]
@richardblack-Harness do you still need this one?

**Comment by richardblack-Harness** [comment]
> @richardblack-Harness do you still need this one?

@ravilach Our roadmap items are in place for Q3 and beyond, I will close this.

**Pull request closed without merging**